book/security.rst

```diff
@@ -275,6 +275,11 @@ But who can you login as? Where do users come from?
     What other methods are supported? See the :doc:`Configuration Reference </reference/configuration/security>`
     or :doc:`build your own </cookbook/security/custom_authentication_provider>`.
 
+.. tip::
+
+    If your application logs in users via a third-party service such as Google,
+    Facebook or Twitter, check out the `HWIOAuthBundle`_ community bundle.
```

Comment:
this bundle is abotu OAuth authentication. this is not specific to Twitter, Facebook or Google only. The wording should be improved